supplementary_style_guide/glossary_terms_conventions/general_conventions/s.adoc

```diff
@@ -218,6 +218,17 @@
 
 *See also*:
 
+[discrete]
+[[SmartNIC]]
+==== image:images/yes.png[yes] SmartNIC
+*Description*: A type of network interface controller (NIC) that uses its own integrated processor to handle certain low-level networking tasks.
+
+*Use it*: yes
+
+*Incorrect forms*: smart NIC
```

Comment:
Please add "Smart-NIC" as an incorrect form.  I spotted it in some code in the OVN-K repo.